packages/react/ds/src/card/card.tsx

```diff
@@ -42,24 +67,43 @@ export const Card = (props: CardProps) => {
           </a>
         </div>
       )}
-      <div className="gi-card-content">
-        <div className="gi-card-heading">
-          <Heading as="h5" customClasses="!gi-my-0">
-            {href ? <Link href={href}>{title}</Link> : title}
-          </Heading>
+      <div className={`gi-card-content ${getInsetClass()}`}>
+        <div className="gi-card-header">
+          <div className="gi-card-heading">
+            <Heading
```

Comment:
Use the heading classes, but don't use the headings tags (eg `e5`) because we don't know what will be the hierarchy of the content.